packages/quantic/force-app/main/default/lwc/quanticResultsPerPage/quanticResultsPerPage.js

```diff
@@ -51,7 +51,13 @@ export default class QuanticResultsPerPage extends LightningElement {
     return this._initialChoice;
   }
   set initialChoice(value) {
-    this._initialChoice = value;
+    if (Number(value) >= 0) {
+      this._initialChoice = value;
+    } else {
+      console.warn(
+        `The "initialChoice" option value "${value}" must be a number greater than 0. Defaulting to the first choices value.`
```

Comment:
To stay consistent in the message logged I would either use the expression `must be a number greater than 0` always or `must be a positive number ` always, currently we use both in the same time.

Reply:
Changed it